package.json

```diff
@@ -29,6 +29,7 @@
     "@reach/router": "^1.2.1",
     "@types/jest": "^24.0.13",
     "@types/jquery": "^3.3.29",
+    "@types/node": "^12.0.4",
```

Comment:
We do not need `@types/node` because we're not going to use `require()`. We should `yarn remove @types/node`

Reply:
done